src/less/lib/app.less

```diff
@@ -5,7 +5,6 @@
   width: 100%;
   background: @app-bg;
   overflow: hidden;
-  .translate3d(0,0,0);
```

Comment:
Just curious, why is that better?

Reply:
The translate3d is used to force use of the GPU, but a side-effect of this is that the sidebar-right won't open on a desktop screen.

Reply:
I kinda can't reproduce it. It would be better to confirm it before. I'd say is at least a distinct bug.